core/models/access.py

```diff
@@ -185,6 +185,15 @@ def clean(self):
         help_text="Was the entry generated automatically, e.g. by REMS?",
     )
 
+    application_id = models.IntegerField(null=True, help_text="REMS application ID.")
```

Comment:
Are these mandatory?  blank =False will force the user to submit them when they create a new access instance

Reply:
done